dallinger/experiment.py

```diff
@@ -590,7 +594,7 @@ def end_experiment(self):
             HerokuApp(self.app_id).destroy()
         return True
 
-    def events_for_replay(self, session=None):
+    def events_for_replay(self, session=None, target=None):
```

Comment:
Are we sure that it's safe to change the interface of this method?

Reply:
I think it is, the only implementations of this method that I can find are the core one, the one in Griduniverse and one in the tests. My feeling is that it's reasonable to assume that the replay feature isn't widely used at this point, so wouldn't be worth doing a major revision dance over.